dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/tracking/ViewTreeChangeTrackingStrategy.kt

```diff
@@ -0,0 +1,42 @@
+package com.datadog.android.rum.internal.tracking
+
+import android.app.Activity
+import android.view.ViewTreeObserver
+import com.datadog.android.rum.GlobalRum
+import com.datadog.android.rum.tracking.ActivityLifecycleTrackingStrategy
+import com.datadog.android.rum.tracking.TrackingStrategy
+
+internal class ViewTreeChangeTrackingStrategy :
+    ActivityLifecycleTrackingStrategy(),
+    TrackingStrategy,
+    ViewTreeObserver.OnGlobalLayoutListener {
+
+    // region ActivityLifecycleTrackingStrategy
+
+    override fun onActivityStarted(activity: Activity) {
+        super.onActivityStarted(activity)
+
+        val viewTreeObserver = activity.window
```

Comment:
We need to check the winow and decorView for nullability here. There will be cases when these can be null and will cause Runtime crashes.